src/Features/Core/Portable/AddImport/IAddImportFeatureService.cs

```diff
@@ -13,6 +14,11 @@
 
 namespace Microsoft.CodeAnalysis.AddImport
 {
+    [DataContract]
+    internal readonly record struct AddImportOptions(
+        [property: DataMember(Order = 0)] bool SearchReferenceAssemblies,
+        [property: DataMember(Order = 1)] bool HideAdvancedMembers);
```

Comment:
much cleaner.  the more you do this the more i like that thsi makes the options the feature use more obvious.